test/TestGrains/MethodInterceptionGrain.cs

```diff
@@ -97,6 +98,53 @@ public Task<object> Invoke(MethodInfo methodInfo, InvokeMethodRequest request, I
         public Task<string> GetInputAsString(bool input) => Task.FromResult(input.ToString(CultureInfo.InvariantCulture));
 
         public Task<int> GetBestNumber() => Task.FromResult(38);
+    }
+
+    public class GrainCallFilterTestGrain : Grain, IGrainCallFilterTestGrain, IGrainCallFilter, IGrainInvokeInterceptor
+    {
+        private const string Key = GrainCallFilterTestConstants.Key;
+        private IGrainCallContext context;
+
+        public async Task<string> Execute(bool early, bool mid, bool late)
```

Comment:
grain method named called Execute is hard to immediately know if this is related to interception or not, since Invoke and Execute both sound like infrastructure methods. Maybe rename to `DoStuff` or something that's clear it's a domain-level method